nsds_lab_to_nwb/metadata/exp_note_reader.py

```diff
@@ -157,4 +198,7 @@ def get_nsds_meta(self):
             self.read_input()
             self.merge_meta_block()
         return self.nsds_meta
-        
+
+path = '/home/jhermiz/Desktop/test'
+reader = ExpNoteReader(path, 'RVG16_B01')
+reader.dump_yaml(write_path=path)
```

Comment:
These lines should be removed. Two alternatives:
- just remove from here and add a test script under `tests/`
- if you want to be able to test run this module like a script, wrap these lines in an `if __name__ == '__main__':` … block

Reply:
My bad. I meant to delete these lines before pushing. I just use this method for local testing